gammapy/catalog/tests/test_hawc.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+from ..hawc import SourceCatalog2HWC
+
+@pytest.fixture #TODO: add here a scope like in test_gammacat.py?
+def hwc2_fhl():
+    filename='$GAMMAPY_EXTRA/datasets/catalogs/2HWC.ecsv' #TODO: Not sure whether this makes sence, __init__ of SourceCatalog2HWC 
+                                                          # has this string as default argument
+    return SourceCatalog2HWC(filename = filename) 
+
+#TODO: Add here @requires_data?
```

Comment:
Yes, you need a `@requires_data('gammapy-extra')` here.
To see it fail locally without adding that, you can do
```
export GAMMAPY_EXTRA=""
```
and then this test will fail when trying to read the ECSV file.
If you put the requires_data decorator the test will be skipped.

Reply:
It works, but if GAMMAPY_EXTRA=""
then the tests are only skipped, they do not return an error?

Reply:
Yip. That's what the `requires_data` decorator is for. Skip test if required test data is not available, don't error on that test.